package.json

```diff
@@ -19,21 +19,10 @@
       "github": "https://github.com/AllMightySauron"
     }
   ],
-  "main": "index.js",
+  "main": "src/index.js",
   "engines": {
     "node": ">=11.14.0"
   },
-  "scripts": {
```

Comment:
Why did you remove de QA dev dependencies and scripts for **unit testing** and **coverage**?